xiangqi_lite/src/components/Button.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+function Button({ children, onClick, className }) {
```

Comment:
We should take `props` only in params and should parse it within components.

All the components/functions should be arrow functions !

Reply:
Also, we are missing propTypes. I would suggest atleast study about it or try to implement in few of the components or through out!